lib/src/migrators/division.dart

```diff
@@ -65,6 +73,49 @@ class _DivisionMigrationVisitor extends MigrationVisitor {
     }
   }
 
+  /// Allows division within a function call's arguments, with special handling
+  /// for new-syntax color functions.
+  @override
+  void visitFunctionExpression(FunctionExpression node) {
+    visitInterpolation(node.name);
+    if (node.name.asPlain == "rgb" ||
+        node.name.asPlain == "rgba" ||
+        node.name.asPlain == "hsl" ||
+        node.name.asPlain == "hsla") {
+      Expression channels;
+      if (node.arguments.positional.length == 1 &&
+          node.arguments.named.isEmpty) {
+        channels = node.arguments.positional.first;
+      } else if (node.arguments.positional.isEmpty &&
+          node.arguments.named.containsKey(r'$channels') &&
+          node.arguments.named.length == 1) {
+        channels = node.arguments.named[r'$channels'];
+      }
+      if (channels != null &&
+          channels is ListExpression &&
+          !channels.hasBrackets &&
+          channels.separator == ListSeparator.space &&
+          channels.contents.length == 3 &&
+          channels.contents[2] is BinaryOperationExpression) {
+        var red = channels.contents[0];
```

Comment:
Using `red` *et al* is a little confusing, because this same path is used for `hsl()` colors. Maybe `channel1` *et al* instead?